receiver/http.go

```diff
@@ -192,6 +198,43 @@ func (htt *HTTP) Start() error {
 	return skogul.Error{Reason: "Shouldn't reach this"}
 }
 
+// verifyPeerCertificate verifies a client certificate presented to us
+// during TLS handshake by comparing its extensions (such as SAN) to
+// some expected value(s)
+func (htt *HTTP) verifyPeerCertificate(_ [][]byte, verifiedChains [][]*x509.Certificate) error {
+	certLogger := httpLog
+	certLogger.Tracef("Verifying %d certificate chains", len(verifiedChains))
+	for _, chain := range verifiedChains {
+		if len(chain) < 1 {
+			// Chain has no certificates (?)
+			continue
+		}
+		cert := chain[0]
+		certLogger := certLogger.WithFields(log.Fields{
```

Comment:
I am not a big fan of overshadowing variable names O_o. The logger isn't a big issue, but imagine trying to debug this at 3AM and not understanding why certLogger means something different on line 206 and line 222.